response/standard_handler.go

```diff
@@ -121,6 +131,32 @@ func (h *standardHandler) Handle(w http.ResponseWriter, r *http.Request, data in
 	}
 }
 
+func (*standardHandler) getData(data interface{}) interface{} {
+	if data == nil {
+		return nil
+	}
+	if v, ok := data.(*standardHandlerDataFieldAny); ok {
+		return v.data
+	}
+
+	reflectType := reflect.TypeOf(data)
+	reflectValue := reflect.Indirect(reflect.ValueOf(data))
+	if reflectType.Kind() == reflect.Ptr {
+		reflectType = reflectType.Elem()
+	}
+	if reflectType.Kind() != reflect.Struct || reflectType.NumField() != 1 {
+		return data
+	}
+	field := reflectValue.Field(0)
```

Comment:
what's the first field of the data? why just judge the first one enough?

Reply:
If there is more than one field, it will not be processed here, and the original one will be returned.